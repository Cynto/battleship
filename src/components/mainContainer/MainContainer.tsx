import React, { useEffect, useState } from 'react';
import Gameboard from '../gameboard/Gameboard';
import './mainContainer.css';
import { useSelector, useDispatch } from 'react-redux';
import EndScreen from '../endScreen/EndScreen';

function MainContainer() {
  const playerGridArray = useSelector((state: any) => state.playerGridArray);
  const AIGrid = useSelector((state: any) => state.AIGrid);
  const playerTurn = useSelector((state: any) => state.playerTurn);
  const [whoWon, setWhoWon] = useState('');

  useEffect(() => {
    let playerBoatsSunk = 0;
    let AIBoatsSunk = 0;
    for (let i = 0; i < playerGridArray.length; i += 1) {
      if (playerGridArray[i] === 7) {
        playerBoatsSunk += 1;
      }
      if (AIGrid[i] === 5) {
        AIBoatsSunk += 1;
      }
    }
    if (playerBoatsSunk === 20) {
      setWhoWon('AI')
    } else if (AIBoatsSunk === 20) {
      setWhoWon('player')
    }
  }, [[playerGridArray, AIGrid]]);

  return (
    <div data-testid="main-container" className="main-container">
      {whoWon === '' ? (
        <div className="grid-turn-container">
          <Gameboard playerGrid={true} gridArray={playerGridArray} />
          <div className="turn">
            <h1>{playerTurn ? 'Your Turn!' : 'AI Turn'}</h1>
          </div>
          <Gameboard playerGrid={false} gridArray={AIGrid} />{' '}
        </div>
      ) : null}
      {whoWon !== '' ? <EndScreen whoWon={whoWon} setWhoWon={setWhoWon}/> : null}
    </div>
  );
}

export default MainContainer;
