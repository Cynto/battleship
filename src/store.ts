import allReducers from './reducers/index';
import { compose, createStore } from 'redux';

declare global {
  interface Window {
    __REDUX_DEVTOOLS_EXTENSION_COMPOSE__?: typeof compose;
  }
}

const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;

const store = createStore(allReducers, composeEnhancers());
export default store