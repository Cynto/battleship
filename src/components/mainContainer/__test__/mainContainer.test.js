import React from 'react';
import ReactDOM from 'react-dom';
import MainContainer from '../MainContainer';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import { HashRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../../../store';

describe('Main Container tests', () => {
  afterEach(cleanup);
  test('Renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
      <Provider store={store}>
        <Router>
          <MainContainer />{' '}
        </Router>
      </Provider>,
      div,
    );
  });
  test('Renders main container correctly', () => {
    const { getByTestId } = render(
      <Provider store={store}>
        <Router>
          <MainContainer />{' '}
        </Router>
      </Provider>,
    );
    expect(getByTestId('main-container')).toHaveClass('main-container');
  });
  test('Matches snapshot', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Router>
            <MainContainer />{' '}
          </Router>
        </Provider>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
