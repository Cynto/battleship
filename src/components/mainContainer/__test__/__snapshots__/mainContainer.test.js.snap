// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Container tests Matches snapshot 1`] = `
Array [
  <div
    className="main-container"
    data-testid="main-container"
  >
    <div
      className="grid-turn-container"
    >
      <div
        className="gameboard-container"
        data-testid="gameboard"
      >
        <div
          className="title-container"
        >
           
          <h2>
            Player Board
          </h2>
        </div>
        <div
          className="grid-container"
        >
          <div
            className="not-visible grid-item"
            data-testid="grid-item-0"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-1"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-2"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-3"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-4"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-5"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-6"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-7"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-8"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-9"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-10"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-11"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-12"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-13"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-14"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-15"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-16"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-17"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-18"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-19"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-20"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-21"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-22"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-23"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-24"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-25"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-26"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-27"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-28"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-29"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-30"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-31"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-32"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-33"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-34"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-35"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-36"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-37"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-38"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-39"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-40"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-41"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-42"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-43"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-44"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-45"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-46"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-47"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-48"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-49"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-50"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-51"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-52"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-53"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-54"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-55"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-56"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-57"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-58"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-59"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-60"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-61"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-62"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-63"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-64"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-65"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-66"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-67"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-68"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-69"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-70"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-71"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-72"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-73"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-74"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-75"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-76"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-77"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-78"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-79"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-80"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-81"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-82"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-83"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-84"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-85"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-86"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-87"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-88"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-89"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-90"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-91"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-92"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-93"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-94"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-95"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-96"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-97"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-98"
          />
          <div
            className="not-visible grid-item"
            data-testid="grid-item-99"
          />
        </div>
      </div>
      <div
        className="turn"
      >
        <h1>
          Your Turn!
        </h1>
      </div>
      <div
        className="gameboard-container"
        data-testid="gameboard"
      >
        <div
          className="title-container"
        >
           
          <h2>
            AI Board
          </h2>
        </div>
        <div
          className="grid-container"
        >
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-0"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-1"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-2"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-3"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-4"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-5"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-6"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-7"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-8"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-9"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-10"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-11"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-12"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-13"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-14"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-15"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-16"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-17"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-18"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-19"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-20"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-21"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-22"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-23"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-24"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-25"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-26"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-27"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-28"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-29"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-30"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-31"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-32"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-33"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-34"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-35"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-36"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-37"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-38"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-39"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-40"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-41"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-42"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-43"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-44"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-45"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-46"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-47"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-48"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-49"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-50"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-51"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-52"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-53"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-54"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-55"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-56"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-57"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-58"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-59"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-60"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-61"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-62"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-63"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-64"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-65"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-66"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-67"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-68"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-69"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-70"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-71"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-72"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-73"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-74"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-75"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-76"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-77"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-78"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-79"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-80"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-81"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-82"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-83"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-84"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-85"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-86"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-87"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-88"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-89"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-90"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-91"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-92"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-93"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-94"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-95"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-96"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-97"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-98"
            onClick={[Function]}
          />
          <div
            className="not-visible grid-item computer-grid"
            data-testid="grid-item-99"
            onClick={[Function]}
          />
        </div>
      </div>
       
    </div>
  </div>,
  " ",
]
`;
